ledger/src/blockstore_db.rs

```diff
@@ -640,6 +865,18 @@ impl Column for columns::TransactionStatus {
     }
 }
 
+impl ColumnMetrics for columns::TransactionStatus {
+    fn report_cf_metrics(
+        cf_metrics: BlockstoreRocksDbColumnFamilyMetrics,
+        column_options: &Arc<LedgerColumnOptions>,
+    ) {
+        cf_metrics.report_metrics(rocksdb_metric_header!(
+            "blockstore_rocksdb_cfs",
+            "transaction_status",
```

Comment:
Ugh, I thought I had something that'd allow only implementing this once as the default implementation in trait definition but didn't pan out. Don't think either of us love this, but also don't see any way around it

Reply:
Exactly, originally (and ideally) I thought I can use the C::NAME or something, but the datapoint macro only accepts literal for the name.  The best solution is still to improve the datapoint macro to allow an arbitrary number of tags, but so far I just don't have a good way.

Reply:
But at least this PR moves this part inside LedgerColumn so that it's cleaner outside, and it also makes it easier to change if we have a better solution.